easybuild/easyconfigs/t/TomoPy/TomoPy-1.11.0-foss-2020b.eb

```diff
@@ -0,0 +1,53 @@
+# Authors: Joachim Hein, Zdenek Matej, Lund University
+
+easyblock = 'PythonPackage'
+
+name = 'TomoPy'
+version = '1.11.0'
+
+github_account = 'tomopy'
+homepage = "https://tomopy.readthedocs.io/en/latest/"
+description = """
+Tomographic reconstruction creates three-dimensional views of an object by 
+combining two-dimensional images taken from multiple directions, for example, 
+this is how a CAT (computer-aided tomography) scanner generates 3D 
+views of the heart or brain.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('scikit-image', '0.18.1'),
+    ('OpenCV', '4.5.1', '-contrib'),
+    ('imkl', '2020.4.304'),
+    ('numexpr', '2.8.1'),
+    ('DXchange', '0.1.8'),
+    ('PyWavelets', '1.1.1'),
+]
+
+builddependencies = [
+    ('scikit-build', '0.11.1'),
+    ('CMake', '3.18.4'),
+    ('Ninja', '1.10.1'),
+]
+
+use_pip = True
+download_dep_fail = True
+sanity_pip_check = True
+
+sources = [{
```

Comment:
I would prefer the more conventional order sources -> builddeps -> deps

Reply:
I am not sure I fully understand this.  I discussed in the slack that TomoPy will not build from a github source.   One needs a full fledged clone.   I you try from a github source you get an error message that suggest you to build from a clone.

@Micket Let me know if I didn't understand your request.

Reply:
I'm talking about the order of things in this file.